docs/elasticsearch.md

```diff
@@ -0,0 +1,204 @@
+# Elasticsearch
+
+With Inago, it's a piece of cake to deploy an Elasticsearch cluster,
+as well as updating it without any downtime or data loss.
+
+It should be noted that this is a demo, and there are dangers to using it in a production environment without further testing.
+Saying that, if you want to do so, and have questions, the team is always available on IRC (`#giantswarm` on `freenode`) <3
+
+## Setup
+- We're using a CoreOS cluster running locally, with Vagrant. See https://coreos.com/os/docs/latest/booting-on-vagrant.html for further information.
+- There are three nodes in the cluster, and each node has 2048MB of memory allocated to it.
+- The `inago` binary, as well as the `elasticsearch` directory from `example`, have been copied to all the nodes in the cluster.
+
+## Starting the Elasticsearch cluster
+We're going to use Inago's `up` command to bring up multiple group slices of the `elasticsearch` group. `up` is equivalent to `submit`, followed by `start`.
+
+Due to the `Conflicts` statement in the unit file for the group, each group slice will be automatically scheduled onto separate machines.
```

Comment:
I'd say `onto separate machines` -> `onto a separate machine` (we're talking about "each group slice")